tests/release/release.go

```diff
@@ -0,0 +1,116 @@
+package release
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	gitopsv1alpha1 "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	klog "k8s.io/klog/v2"
+	"knative.dev/pkg/apis"
+)
+
+var snapshotComponents = []gitopsv1alpha1.ApplicationSnapshotComponent{
+	{"component-1", "quay.io/redhat-appstudio/component1@sha256:d5e85e49c89df42b221d972f5b96c6507a8124717a6e42e83fd3caae1031d514"},
+	{"component-2", "quay.io/redhat-appstudio/component2@sha256:a01dfd18cf8ca8b68770b09a9b6af0fd7c6d1f8644c7ab97f0e06c34dfc5860e"},
+	{"component-3", "quay.io/redhat-appstudio/component3@sha256:d90a0a33e4c5a1daf5877f8dd989a570bfae4f94211a8143599245e503775b1f"},
+}
+
+var _ = framework.ReleaseSuiteDescribe("test-release-service-happy-path", Label("release"), func() {
+	defer GinkgoRecover()
+	// Initialize the tests controllers
+	framework, err := framework.NewFramework()
+	Expect(err).NotTo(HaveOccurred())
+
+	var devNamespace = uuid.New().String()
+	var managedNamespace = uuid.New().String()
+
+	BeforeAll(func() {
+		// Create the dev namespace
+		demo, err := framework.CommonController.CreateTestNamespace(devNamespace)
+		Expect(err).NotTo(HaveOccurred(), "Error when creating namespace '%s': %v", demo.Name, err)
+
+		// Create the managed namespace
+		namespace, err := framework.CommonController.CreateTestNamespace(managedNamespace)
+		Expect(err).NotTo(HaveOccurred(), "Error when creating namespace '%s': %v", namespace.Name, err)
+
+		// Wait until the "pipeline" SA is created and ready with secrets by the openshift-pipelines operator
+		klog.Infof("Wait until the 'pipeline' SA is created in %s namespace \n", managedNamespace)
+		Eventually(func() bool {
+			sa, err := framework.CommonController.GetServiceAccount("pipeline", managedNamespace)
+			return sa != nil && err == nil
+		}, 1*time.Minute, defaultInterval).Should(BeTrue(), "timed out when waiting for the \"pipeline\" SA to be created")
+	})
+
+	AfterAll(func() {
+		// Delete the dev and managed namespaces with all the resources created in them
+		Expect(framework.CommonController.DeleteNamespace(devNamespace)).NotTo(HaveOccurred())
+		Expect(framework.CommonController.DeleteNamespace(managedNamespace)).NotTo(HaveOccurred())
+	})
+
+	var _ = Describe("Creation of the 'Happy path' resources", func() {
+		It("Create an ApplicationSnapshot.", func() {
```

Comment:
Integration Service would have a need to validate that snapshots can be created as well. It is a little odd that the snapshot test would be tied into the release happy path suite. Based on the description I would assume you would be validating that you can create the resources you own or directly create even if they aren't apart of the API? 

I wonder it should move up into the BeforeAll? Although this does make me questions if we need to start labeling tests like these as sanity/smoke suite

Reply:
In fact, 
this is about the e2e-test (happy-path) from milestone 5, 
when our team merged the changes been made towards demo 7, this test was broken and other tests, 
thus we removed disabled e2e tests to make the merge happen and now we are working on enable back the e2e tests, 
just it require refactoring the test flow and for sure the CRDs creating with the validation part (post CRDs), 
in order to fix it ASAP, we saw it's much efficient that  we divide this e2e test into small parts so I and Nafta can work in parallel . This is part of the test